### PR TITLE
Skeleton phpBB 3.3 Updates

### PR DESCRIPTION
**Pull request by VSEphpbb**
The skeleton ext is no longer supported being installed to a 3.1.x board. It had some composer dependencies that can be removed, as they were only in place for use on 3.1.x boards.

Also made changes to support phpBB 3.3.x extension builds

### CODE REVIEW

File: ext.php [hunk 1]
```diff
@@ -15,38 +15,80 @@
...
+	 * Check PHP 5.4.0 minimum requirement.
 	 *
-	 * @return bool
+	 * @return void
 	 */
-	protected function php_requirements()
+	protected function php_requirement()
 	{
-		return phpbb_version_compare(PHP_VERSION, '5.4.0', '>=') && class_exists('ZipArchive');
+		if (phpbb_version_compare(PHP_VERSION, '5.4.0', '<'))
```
Comment (rxu) [inline ext.php]: It won't be installed on phpBB < 3.2, but 3.2 can be only installed with PHP 5.4.7+.
So this check is probably can be dropped here as it is always satisfied.

File: language/en/common.php [hunk 2]
```diff
@@ -144,4 +144,8 @@
 	'SKELETON_INVALID_NUM_AUTHORS'		=> 'The number of authors you provided is invalid',
 	'SKELETON_INVALID_PACKAGE_NAME'		=> 'The package name you provided is invalid',
 	'SKELETON_INVALID_VENDOR_NAME'		=> 'The vendor name you provided is invalid',
+
+	'NO_ZIPARCHIVE_ERROR'	=> 'The ZipArchive class is required, but was not found in your PHP configuration.',
+	'PHP_VERSION_ERROR'		=> 'PHP 5.4 or newer is required to use this extension.',
```
Comment (rxu) [inline language/en/common.php]: Also not needed if PHP version check is dropped (above).

File: ext.php (hunk 1 above)
Comment (3D-I) [reply ext.php]: PHP 5.4.7 is due to the s9e library which updated at some point, previous 3.2 versions run on 5.4.0.

Comment (VSEphpbb) [reply ext.php]: Also, it doen't hurt just to check :)

**Pull request merged**